src/modules/limitOrders/containers/TradeButtons/index.tsx

```diff
@@ -53,6 +54,9 @@ export function TradeButtons(props: TradeButtonsProps) {
     doTrade: handleTrade,
     quote,
     isSupportedWallet,
+    wrapNativeFlow() {
+      wrapNativeFlow()
+    },
```

Comment:
Since it's exactly the same name, can't you:

```suggestion
    wrapNativeFlow,
```

Reply:
Fixed